### PR TITLE
 JAMES-2329 Communication for Google Summer of Code 2018 

### PR DESCRIPTION
**Pull request by chibenwa**
capture d ecran de 2018-02-02 17-33-53
capture d ecran de 2018-02-02 17-34-07

### CODE REVIEW

File: src/homepage/_posts/2018-02-12-gsoc-2018.markdown [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title:  "Apache JAMES joins Google Summer of code 2018 !!!"
```
Comment (rouazana) [inline src/homepage/_posts/2018-02-12-gsoc-2018.markdown]: no triple ! please :)

File: src/homepage/_posts/2018-02-12-gsoc-2018.markdown [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+date:   2018-02-1 00:00:22 +0200
+categories: announcements
+---
+
+This year, James developers will propose ideas of Google Summer of Code projects for students.
+
+[Google Summer of Code] is a global program focused on bringing more student developers into open source software development.
+Students work with an open source organization on a 3 month programming project during their break from school.
+
+![My helpful screenshot]({{ "/assets/images/gsoc.png" | absolute_url }})
```
Comment (rouazana) [inline src/homepage/_posts/2018-02-12-gsoc-2018.markdown]: better naming?

File: src/homepage/_posts/2018-02-12-gsoc-2018.markdown [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+Here are some ideas of applications:
+
+ - ⚒ [JAMES-1931] We are in need of a WebAdmin User Interface to ease our beloved administrator API. You will need to build an administration website on top of the **webAdmin** REST API.
+
+ - ⚒ [JAMES-1932] James allow to fully customize mail processing. But we miss tools to ease edition and testing of the processing flow.
+
+ - ⚒ [JAMES-1933] Emails need to be secure. Many protocols like DKIM, SPF and DMARC allow to meet this purpose. However we need more documentation and a DMARC implementation.
+
+ - ⚒ [JAMES-XXXX] James should be a flexible server. We should be able to use custom configuration providers, not only XML files as off today. That is a move toward "clustering done the right way".
```
Comment (rouazana) [inline src/homepage/_posts/2018-02-12-gsoc-2018.markdown]: now you can replace XXX
s/off today/of today/

File: src/homepage/_posts/2018-02-12-gsoc-2018.markdown (hunk 2 above)
Comment (chibenwa) [reply src/homepage/_posts/2018-02-12-gsoc-2018.markdown]: For sure...

**Comment by chibenwa** [comment]
Merged, and communication done ;-)

**Pull request closed without merging**